audit/hashstructure.go

```diff
@@ -11,10 +11,11 @@ import (
 	"time"
 
 	"github.com/hashicorp/go-secure-stdlib/strutil"
-	"github.com/hashicorp/vault/sdk/helper/wrapping"
-	"github.com/hashicorp/vault/sdk/logical"
 	"github.com/mitchellh/copystructure"
 	"github.com/mitchellh/reflectwalk"
+
```

Comment:
Can you remove this blank line please?

Reply:
Do we have a style guide or import formatter that we're using? I've been using `goimports` with grouping by
1. stdlib
2. 3rd party packages
3. 1st party packages

but i'm happy to change this so we're all consistent.

Reply:
Upon further investigation, it looks like we're using gofumpt, which only groups by
1. stdlib
2. non stdlib

I'll make this change!